src/microbe_stage/editor/CellEditorComponent.GUI.cs

```diff
@@ -428,6 +428,9 @@ private void UpdateGUIAfterLoadingSpecies(Species species, ICellProperties prope
         UpdateGeneration(species.Generation);
         UpdateHitpoints(CalculateHitpoints());
         UpdateStorage(CalculateStorage());
+
+        // Set the editor light level and associated GUI elements to daytime
+        SetLightLevelOption("Day");
```

Comment:
This should do something like `LightLevelOption.Day.ToString()` for futureproofing.